runtime/tests/port/si.c

```diff
@@ -2280,6 +2281,27 @@ j9sysinfo_test_get_l1dcache_line_size(struct J9PortLibrary *portLibrary)
 	return reportTestExit(portLibrary, testName);
 }
 
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+I_32
```

Comment:
dont mix omr type and stdint. I would simply follow the convention in this file

Reply:
Would that be to replace `uintptr_t pid` with `U_64 pid`, `uint64_t processStartTimeInNanoseconds` with `U_64 processStartTimeInNanoseconds`, and `int32_t rc` with `I_32 rc`?

Reply:
uintptr_t - > UDATA, the rest are correct